README.md

```diff
@@ -80,7 +80,7 @@ Copy the `guildId` from the Discord server you want to add the bot to:
 
 ![guild id](./assets/3-copy-id.png)
 
-> Note: In order to copy a guild id, you will need to have Discord developer mode enabled, scroll down in the guide to see how to do so (its easy!)
+> Note: In order to copy a guildId, you will need to have Discord developer mode enabled, scroll down in the guide to see how to do so (it's easy!)
```

Comment:
```suggestion
> Note: In order to copy a `guildId`, you will need to have Discord developer mode enabled, scroll down in the guide to see how to do so (it's easy!)
```

Reply:
If the intent of this fix is to emphasize that the guild id is the guildId from the image above we should also emphasize it with code formatting